doublyLinkedList.js

```diff
@@ -168,6 +168,52 @@ class DoublyLinkedList {
     }
   }
 
+  sortByAsc() {
```

Comment:
I implemented another method of sorting for linked list. I think it's a little bit better than first sort implementation. Also I found a topic on the Wikipedia. In topic described method of sorting linked list, but I don't understand it yet. (In described method we have to use stack with stack element wrapper. It stores node and level)